Sources/App/RepoMapping.swift

```diff
@@ -3,24 +3,40 @@ import Vapor
 
 struct RepoMapping {
     let repository: GitHubService.Repository
-    let environment: JiraService.CRPIssueFields.Environment
-    let jiraSummary: (GitHubService.Release) -> String
+    let crp: CRP
+
+    struct CRP {
```

Comment:
seems like on Linux there are some issues with tuple types that I tried instead of this struct first